### DISCUSSION TITLE
LFX Mentorship Mentee Applications for '23 Term 03 (September - November)

### DISCUSSION THREAD

**Discussion by nate-double-u** [cncf/mentoring#1066] [Announcements]
Hello everyone,

The LFX mentorship projects for 2023 Term 03 (September - November) are now listed publicly, and can be found here:
https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2023/03-Sep-Nov

Mentee applications are now open!

Applications will be open until Tues Aug 15, 5:00 PM (PDT).

We encourage everyone interested to apply and engage with the project community and mentors.

# Mentee Timeline: Aug 2nd - Nov 30th

## Applications

* Aug 2nd - Aug 15th, 2023 24:00 UTC / 5:00 PM PDT (2 weeks): Mentee applications open
* Aug 16th - Aug 29th: Application review by the mentors, admission decisions

## Mentorship duration - three months (12 weeks - full-time schedule)

* Sept 4, 2023 (Week 1): Mentorship program begins with the initial work assignments
* Oct 11, 2023 (End of Week 6): Midterm mentee evaluations and first stipend payments
* Nov 22, 2023 24:00 UTC / 5:00 PM PDT (End of Week 12): Final mentee evaluations and mentee feedback/blog submission due, second and final stipend payment approvals
* Nov 30: Last day of term

**Comment by anirudhprabhakaran3** [comment]
Hi! Just a question. These projects are the same as those listed on the mentorship site, correct? Like we can refer to either one of the two for information?

**Reply by aliok** [reply]
hi @anirudhprabhakaran3 
Yes, they are the same.

**Reply by anirudhprabhakaran3** [reply]
Thank you!

**Comment by jaydee029** [comment]
Hi, I would like to participate in LFX mentorship. How can I request to add my email to volcano and/or cloud-native slack channel. or do we use any other means of communication?
Thanks
email-dhruvj797@gmail.com

**Reply by aliok** [reply]
hello @jaydee029 

Please check the communication information of the project website/repository.

**Comment by satyampsoni** [comment]
Hi @aliok. what is the date for the announcement of results?

**Reply by aliok** [reply]
@satyampsoni do you see your name in the project you've applied for? In https://mentorship.lfx.linuxfoundation.org/?

I was on PTO, so I will refer to @nate-double-u here

**Reply by nate-double-u** [reply]
We've been going through and doing acceptances in the LFX Platform. We're through most of them now, there are a couple projects still that need to get sorted out, but most people should have been emailed whether or not they have been accepted to the programs they applied to by now.

**Comment by thefool76** [comment]
So the mentee applications are closed?

**Reply by nate-double-u** [reply]
Yes, our next intake starts in [January 2024](https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2024/01-Mar-May).

**Reply by thefool76** [reply]
ok

**Comment by shubham-cmyk** [comment]
Hi @nate-double-u 
I am Shubham Gupta. 
I am facing some mailing address issues is there any way I can communicate with you? 
Is there any slack channel or something?

**Reply by nate-double-u** [reply]
Hi @shubham-cmyk, we've emailed you at the new address you provided. Please reach out to me on the CNCF slack, @nate-double-u, if you need more help).
